src/_kw

```diff
@@ -282,6 +282,7 @@ _kw_explore()
     '(-a --all -l --log -g --grep)'{-a,--all}'[search for all matches in files under or not git management]' \
     '(-c --only-source -H --only-header)'{-c,--only-source}'[show only results from the source]' \
     '(-H --only-header -c --only-source)'{-H,--only-header}'[show only results from the header]' \
+    '(-C --show-context)'{-C,--show-context}'[set the context value]' \
```

Comment:
The short option accepts arguments that come directly after, like in `-C5`, and the long option accepts arguments after the equal sign, like in `--show-context=5`. However, the Zsh completions leave a blank space after completing `-C` and `--show-context,` both of which will result in a parser error and a bad experience for the user. To fix this, change the line to

```bash
'(-C --show-context)'{-C-,--show-context=-}'[set the context value]' \
```

with the minus after `-C` and an equal sign and minus after `--show-context` inside the curly braces.

You probably shouldn't know this if you never played with Zsh completions (which aren't intuitive), so don't worry.